liquidhaskell.cabal

```diff
@@ -195,7 +195,7 @@ library
                     , aeson
```

Comment:
@facundominguez @ranjitjhala we should change the version to `version: 0.9.2.0.0`? The convention is that the first two numbers follow GHC's version

Reply:
In 769a8fce1 I set the version to `0.9.2.5.0`.